ido-vertical-mode.el

```diff
@@ -87,7 +87,7 @@ so we can restore it when turning `ido-vertical-mode' off")
 (defvar ido-vertical-count-active nil
   "Used internally to track whether we're already showing the count")
 
-(defcustom ido-vertical-define-keys 'C-n-and-C-p-only
```

Comment:
The problem is that changing default Emacs bindings can catch users by surprise, even when done with the best intentions.  This is especially a problem with IDO, because its keys are not discoverable (you can't press `C-h m` and see what bindings are available).  Sure, you can document the new key bindings, but will users read the README and remember it?  Indeed, I came here because I kept getting an error when pressing `M-p` and expecting it to call `previous-history-element`, before discovering that it didn't.  Therefore, I think that the default behaviour of IDO Vertical Mode should be to list completions vertically, and nothing else.  The "principle of least surprise" applies here.  The option to use `C-p` and `C-n` is nice to have, though! And, by the way, I have realized that you could remap `ido-toggle-prefix` as `C-c C-t` directly, without asking the user to do anything.

These are my personal thoughts.  If you decide to keep the current default, that will be fine with me because -- thankfully -- you have kindly released IDO Vertical Mode as free software and I can just keep my fork :-)

Cheers.